modules/exploits/example_webapp.rb

```diff
@@ -10,14 +10,19 @@
 #
 ###
 class MetasploitModule < Msf::Exploit::Remote
-  Rank = NormalRanking
+  Rank = NormalRanking # https://github.com/rapid7/metasploit-framework/wiki/Exploit-Ranking
 
   #
   # This exploit affects a webapp, so we need to import HTTP Client
   # to easily interact with it.
   #
   include Msf::Exploit::Remote::HttpClient
 
+  # there are libraries for several CMSes and such like wordpress, typo3, sharepoint, nagios_xi, moodle
+  # joomla jboss, drupal.
+  # https://github.com/rapid7/metasploit-framework/tree/master/lib/msf/core/exploit/remote/http
+  include Msf::Exploit::Remote::HTTP::Wordpress
```

Comment:
```suggestion
  # There are libraries for several CMSes and such like Wordpress, Typo3, Sharepoint, Nagios XI, Moodle,
  # Joomla, JBoss, Drupal, etc. The following code just includes the Wordpress library, however you can
  # find more at
  # https://github.com/rapid7/metasploit-framework/tree/master/lib/msf/core/exploit/remote/http
  include Msf::Exploit::Remote::HTTP::Wordpress
```

Reply:
So long as you're correcting capitalization... Both `WordPress` and `SharePoint` use a capital `P`.

Reply:
Thanks will fix that as well @bcoles 👍

Reply:
Resolved in https://github.com/rapid7/metasploit-framework/pull/15867/commits/fc05f5368860ead75ed6ef34387845f504200009 and capitalization issues in https://github.com/rapid7/metasploit-framework/pull/15867/commits/a620b425ddffe94b8f4a725650d0125359bedbce